src/get-rollup-options.js

```diff
@@ -5,10 +5,14 @@ import req from 'req-cwd'
 import stringifyAuthor from 'stringify-author'
 
 function getDest(options, format, compress) {
-  const filename = options.filename
+  let filename = options.filename
   const dir = options.outDir
   let suffix = ''
 
+  if (filename.startsWith('@')) {
```

Comment:
Looks like a better option to me as well. Are you happy with checking for starting with '@' in [/src/get-config.js@`master`#L8](https://github.com/egoist/bili/blob/master/src/get-config.js#L8)?